app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
+  after_save :posts_count
+  has_many :comments_comments
+  has_many :likes
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def posts_count
+    user.update!(:posts_count)
+  end
+
+  private :posts_count
+end
```

Comment:
- [ ] Great work creating the models! Let's add the missing validations for the `Post` model:

- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.
- LikesCounter must be an integer greater than or equal to zero.

Tip: You can use `validates :Something` for this.